lib/perl/Genome/Qc/Tool/VerifyBamId.pm

```diff
@@ -0,0 +1,59 @@
+package Genome::Qc::Tool::VerifyBamId;
+
+use strict;
+use warnings;
+use Genome;
+use List::MoreUtils qw(uniq);
+
+class Genome::Qc::Tool::VerifyBamId {
+    is => 'Genome::Qc::Tool::WithVariationListVcf',
+};
+
+
+sub cmd_line {
+    my $self = shift;
+
+    my $cmd = $self->gmt_class->create($self->gmt_params);
+    return $cmd->_get_cmd_list;
+}
+
+sub metrics {
+    return (
+        number_snps => '#SNPS',
+        freemix => 'FREEMIX',
+        chipmix => 'CHIPMIX',
+    );
+}
+
+sub get_metrics {
+    my $self = shift;
+
+    my $file = $self->qc_metrics_file . '.selfSM';
+    my $reader = Genome::Utility::IO::SeparatedValueReader->create(
+        input => $file,
+        separator => '\t',
+        is_regex => 1,
```

Comment:
This needn't be `is_regex` if the separator is `"\t"` instead of `'\t'`.  (Since the current implementation uses a regular expression either way, I wonder which one is more performant.)